packages/components/src/avatar/src/svgs.tsx

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable max-len */
+import React from "react";
+
+export const DeletedIcon: React.FunctionComponent = () => (
+    <svg fill="none" height="64" viewBox="0 0 64 64" width="64" xmlns="http://www.w3.org/2000/svg">
+        <path d="M0 32C0 14.3269 14.3269 0 32 0C49.6731 0 64 14.3269 64 32C64 49.6731 49.6731 64 32 64C14.3269 64 0 49.6731 0 32Z" fill="#E5DED6" />
```

Comment:
can we replace fill's hardcoded hex with something else? are those the same color in darkmode? 

in both svg

Reply:
Good question, it didn't have a dark mode version. I suppose if I used a variable, then we wouldn't need to worry. I just copied the icon from figma.
I originally was going to not include the grey outside in the icon and do it using html/css.
Then I thought the sizes of the icons would be weird, plus the anonymous icon's face wasn't grouped together so it wasn't easy exporting...
How do you think it should be done?

Reply:
I think it's ok for the svg to have the grey round background. 
For the other issue, I'd like you to propose something.

Reply:
alright then I'll just replace the color with a css variable. I think it should work

Reply:
Can you validate with Design for the dard mode?

Reply:
The color is in tokens, so it's easy to know what it should be for dark mode. I put the right variable and it'll be good. i can send a screenshot to design once I do it if you prefer.

Reply:
validated and it's good in dark mode